processes/hackathon_sponsorship_process.md

```diff
@@ -0,0 +1,98 @@
+# Hackathon Sponsorship Process
+
+Process for working with a hackathon that reaches out to us for sponsorship.
+
+**Table of Contents**
+
+- [Hackathon Sponsorship Process](#hackathon-sponsorship-process)
+  - [Goals Of Sponsorship For Hack Club](#goals-of-sponsorship-for-hack-club)
+  - [Fields](#fields)
+  - [Stages](#stages)
+    - [Contacted Us](#contacted-us)
+    - [Responded](#responded)
+    - [Invited To Event](#invited-to-event)
+    - [Successful Hackathon](#successful-hackathon)
+    - [Rejected](#rejected)
+
+Hackathon sponsorships should be tracked in the `Hackathons` Streak pipeline. Just like any other Streak pipeline, make sure to link new emails to related Streak Boxes and keep the pipeline up to date when things change.
+
+## Goals Of Sponsorship For Hack Club
+
+We're looking to help spread hacker ethos and technical knowledge at these events.
+
+**We are looking to provide/receive:**
+
+- Mentors
+- Workshops
+- Hack Club's logo on the hackathon's website
+- Talk during opening ceremony
+- Have a table to pass out stickers
+
+**We do not provide:**
+
+- Financial support
+- Company contacts
```

Comment:
If they're a Hack Club would we make an exception here and reach out to our contacts? Seems like a very strong rule.